test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -740,11 +759,20 @@ protected static T WaitForValue<T>(Func<T> act, T expectedVal, int timeout = 150
             } while (true);
         }
 
-        public static void WaitForUserToContinueTheTest(string url, bool debug = true)
+        public static void WaitForUserToContinueTheTest(string url, bool debug = true, X509Certificate2 clientCert = null)
         {
             if (debug && Debugger.IsAttached == false)
                 return;
 
+            if (clientCert != null && RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
please note that you also in finally need to remove this

Reply:
done